lib/attribute-parser.js

```diff
@@ -18,6 +20,9 @@ module.exports = class AttributeParser {
       /^\s+/, // whitespace
       /^[^\s]+/ // word
     ]
+
+    // extend global variables
+    global = Object.assign(global, globalContext)
```

Comment:
This is bad :) if you do this no need to bother to pass it to the constructor, it's like add it directly from your code or from `trailpack-annotations`. I think it can be done with `vm.runInContext(exp, globalContext)` instead of `runInThisContext`?